com.microsoft.mrtk.input/MRTK.Input.asmdef

```diff
@@ -6,8 +6,7 @@
         "Microsoft.MixedReality.Toolkit.Core",
         "Unity.InputSystem",
         "Unity.XR.CoreUtils",
```

Comment:
Should we add `"com.unity.xr.core-utils": "2.0.0"` to the Input package.json as well?

Reply:
Added!